packages/ui/src/tamagui.config.ts

```diff
@@ -67,6 +67,7 @@ export const tokens = createTokens({
     '3xl': 32,
     '4xl': 48,
     '5xl': 64,
+    '6xl': 72,
```

Comment:
Is this shown in design somewhere? Would rather not add here unless we really need it.

Reply:
Yeah, the channel items are shown as 72px: https://www.figma.com/design/mUIcOGeELxWswuusYUnEiW/2024-Q2-Tlon.app?node-id=4001-64568&t=FhFpsvOxwFgP6jzV-4 

Could hardcode as 72px?

Reply:
Ah got it. Yeah, I think hardcode for now -- let's see if it gets designed into the system.